#--------------------------------------------------------#
#--------------- MAPPING    -----------------------------#
#--------------------------------------------------------#

#--------------- ROOMS -------------------------------#
rooms:
    - name : kitchen
      pose1: [5.06,-0.37] #top_left
      pose2: [8.9,-0.76]  #top_right
      pose3: [8.08,-4.43] #bottom_right
      pose4: [4.24,-3.6] #bottom_left
    - name : living_room
      pose1: [4.23,-3.7] #top_left
      pose2: [8.04,-4.54]  #top_right
      pose3: [7.08,-9.88] #bottom_right
      pose4: [3.27,-9.27] #bottom_left
    - name : bedroom
      pose1: [0.8,-4.0] #top_left
      pose2: [4.02,-4.55]  #top_right
      pose3: [3.22,-9.26] #bottom_right
      pose4: [0.14,-8.64] #bottom_left
    - name : bathroom
      pose1: [1.37,-1.18] #top_left
      pose2: [4.6,-1.8]  #top_right
      pose3: [4.01,-4.49] #bottom_right
      pose4: [0.83,-4.0] #bottom_left
    - name : entrance
      pose1: [1.61,0.35] #top_left
      pose2: [3.49,-0.01]  #top_right
      pose3: [3.2,-1.55] #bottom_right
      pose4: [1.37,-1.14] #bottom_left
    - name : corridor
      pose1: [3.52,-0.01] #top_left
      pose2: [5.01,-0.18]  #top_right
      pose3: [4.6,-1.85] #bottom_right
      pose4: [3.26,-1.58] #bottom_left



#--------------- LINKS (DOORS)-----------------------#
connection:
    - name: con1
      rooms: ["kitchen","living_room"]
      door: false
    - name: con2
      rooms: ["kitchen","bathroom"]
      door: true
    - name: con3
      rooms: ["living_room","bedroom"]
      door: true
    - name: con4
      rooms: ["bedroom","bathroom"]
      door: true
    - name: con5
      rooms: ["Unknown","entrance"]
      door: true
    - name: con6
      rooms: ["entrance","corridor"]
      door: true
    - name: con7
      rooms: ["corridor","kitchen"]
      door: true



objects:
    - name: bottle
      seen: false
      mapped: true
      pose: [0.0,0.0,0.0]
      room: ''
      furniture: ''
      expected_rooms: ["kitchen","living_room"] # Ordered to most likely to less likely
      expected_furniture: ["kitchen_table","tv_table"]


furniture:
    - name: kitchen_table
      static: true
      seen: false
      room: kitchen
      pose: [6.4,-2.4]
      orientation: 1.3
      size: [1,0.6,1.0]
    - name: tv_table
      static: true
      seen: false
      room: living_room
      pose: [5.3,-8.3]
      orientation: -0.2
      size: [0.6,0.6,0.4]




#--------------------------------------------------------#
#--------------- NAVIGATION -----------------------------#
#--------------------------------------------------------#
waypoints:
      - name : door_livingroom_to_bedroom
        pose: [4.17,-6.12]
        room: living_room
      - name : door_bedroom_to_livingroom
        pose: [3.31,-6.04]
        room: bedroom
      - name : door_kitchen_to_bathroom
        pose: [4.77,-2.56]
        room : kitchen
      - name : door_bathroom_to_kitchen
        pose: [3.97,-2.39]
        room : bathroom
      - name : door_bedroom_to_bathroom
        pose: [3.14,-4.81]
        room : bedroom
      - name : door_bathroom_to_bedroom
        pose: [3.32, -3.96]
        room : bathroom

